plugins/inputs/win_wmi/win_wmi_test.go

```diff
@@ -68,7 +68,9 @@ func TestWmi_Gather(t *testing.T) {
 	var logger = new(testutil.Logger)
 	var acc = new(testutil.Accumulator)
 	plugin := Wmi{Queries: []Query{testQuery}, Log: logger}
-	plugin.Init()
+	err := plugin.Init()
+	require.NoError(t, err)
```

Comment:
Matter of taste but I prefer
```suggestion
	require.NoError(t, plugin.Init())
```

Reply:
Changed